src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/packages.config

```diff
@@ -1,13 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Xamarin.Android.Arch.Core.Common" version="1.0.0" targetFramework="monoandroid81" />
```

Comment:
was this file supposed to be deleted?

Reply:
Visual Studio deleted it because there are no more packages to install for this project.